components/email/src/Email.svelte

```diff
@@ -936,15 +936,16 @@
               background: none;
               display: flex;
               cursor: pointer;
-
-              * {
-                width: 0.7em;
-                height: 0.7em;
-              }
             }
           }
           [role="toolbar"] {
             outline: none;
+            button {
+              * {
```

Comment:
Hmm - this seems, excessive no? Using `*`. TBH, I am not sure why we did this before as well.

Reply:
Technically, we want this for the icons only, so it makes sense to change this to svg instead

Reply:
IIRC `svg` selector doesn't work for these since the SVGs are loaded as components

Reply:
Apparently it does not apply the style to the svg element like this

Reply:
Ah I see why we did this in the first place, it is because we're using custom elements to render svg and svelte does not parse the :global() selector in custom elements